docs/blog/2019-03-11-dot-org-prototypes/index.md

```diff
@@ -0,0 +1,139 @@
+---
+title: ".org Prototype Test Blogpost"
+date: 2019-03-11
+author: Shannon Soper
+tags: ["ux," "design"]
+---
+
+Five brave souls who had never heard of Gatsby before tested our lo-fi prototype of the gatsbyjs.org mobile homepage last week. Along with help from our designer @fk and many others on the core team, we drafted 7 prototypes and got solid feedback on them.
+
+Here are the [lo-fi prototypes](https://www.figma.com/proto/UH2Qb3IeF8Hvg6csIW3mcqFc/Gatsbyjs.org-mobile-homepage-prototype?node-id=22%3A329&viewport=-227%2C349%2C0.39312&scaling=scale-down), if you’re curious to see what we tested.
+
+## Stats on usability testers
+
+- 6 testers
+- All of them know React
+- 5 of them had never heard of Gatsby before
+
+## Why did we do this usability test?
+
+### We did another usability test in November with bummer results!
+
+Back in November, we watched 8 people who had never heard of Gatsby before as they landed on the homepage and asked them “what is Gatsby” and “how do you decide if you want to use it?” The results were...not as accurate as we hoped!
+
+Two problems became clear as a result:
+
+- The first impression people have about Gatsby from above-the-fold and just below the fold on homepage is not accurate. Sources of more accurate understanding: Github READme file, diagram on homepage
+- We had not written down these key facts about the .org site and therefore sometimes struggled to stay aligned:
+  - Who is the audience for .org?
+  - What are our value propositions for that audience?
+  - What is the purpose of the homepage?
+  - How do we know if we’re succeeding in that purpose?
+
+It is important to the Gatsby community to solve these problems, because tons of new visitors come to the gatsbyjs.org homepage each month. 130,000 in January alone! We want to make sure we’re accurately representing Gatsby’s value there and helping more developers experience Gatsby’s value by clicking the “get started” button.
```

Comment:
This would become a complete sentence with a colon instead of a period: "each month: 130,000 in January alone!"